src/sagas/stats.js

```diff
@@ -43,7 +43,7 @@ export function* callUnitsStats(args, namespace) {
 }
 
 function* watchFetchList() {
-  yield takeLatest([lists.FETCH_LIST, tags.FETCH_STATS], fetchCUStats);
+  yield takeLatest([tags.FETCH_STATS], fetchCUStats);
```

Comment:
Why FETCH_LIST is not relevant any more.

Reply:
stats now not part of list but part of filters. It have special actions